.github/workflows/tox.yml

```diff
@@ -11,7 +11,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - '3.6'
           - '3.7'
```

Comment:
Not sure what OME's position is but we're slowly approaching Python 3.7 EOL (June 2023). One option here would be to also drop 3.7 from the testing matrix (and add 3.10?). This would also remove the need to special case the execution of `flake8` on certain environments in https://github.com/ome/omero-marshal/pull/75/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R22

Reply:
Dropping 3.7 has not been discussed. It certainly creates some issue. See for example https://github.com/ome/omero-cli-server/pull/28
I will add 3.10 for now